index.js

```diff
@@ -540,8 +540,8 @@
   //. The Maybe type represents optional values: a value of type `Maybe a` is
   //. either a Just whose value is of type `a` or a Nothing (with no value).
   //.
-  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][], and
-  //. [Extend][] specifications.
+  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][],
+  //. [Traversable][], and [Extend][] specifications.
```

Comment:
Haha, I looked at this yesterday and couldn't remember why `Foldable` was removed previously and presumed `reduce` hadn't yet landed at that time. That'll teach me for not looking at the comment history :smile: 

In any case, I've removed it again.